tests/toydata/docs_nested.csv

```diff
@@ -0,0 +1,4 @@
+count,text,image,image2.url
```

Comment:
```suggestion
count,text,image,image2__url
```

Not really sure here but I think we cannot/should not use `.` inside column name

Reply:
Yeah kind of because, though it is more gut feeling that actual knowledge about CSV convention. But generally I think we should keep with `a__b` sintax rather than `a.b` one. Same for vector database I believe @JohannesMessner